program/application/Main.java

```diff
@@ -1,7 +1,58 @@
 package application;
 
+import java.util.Scanner;
+
 public class Main {
+
     public static void main(String[] args) {
+        boolean debug = false;                             //debugmodus aus-ein
+        enum gameState {running, deal, end}                //spielstatus
```

Comment:
enum would be better outside, not inside the main method
also, enums should be uppercase